developer-extension/src/panel/components/tabs/infosTab.tsx

```diff
@@ -130,7 +162,47 @@ function Entry({ name, value }: { name: string; value: any }) {
         </>
       ) : value ? (
         <>
-          {name}: <Json value={value} />
+          {name}
+          {onChange && (
+            <>
+              {!edited ? (
+                <ActionIcon
+                  style={{ display: 'inline-block' }}
+                  color="violet"
+                  size="xs"
+                  variant="transparent"
+                  aria-label="Edit"
+                  onClick={() => setEdited(true)}
+                >
+                  ✏️
+                </ActionIcon>
+              ) : (
+                <Button
+                  color="violet"
+                  variant="transparent"
+                  size="compact-xs"
+                  onClick={handleApplyClick}
+                  className="dd-privacy-allow"
+                >
+                  Apply
+                </Button>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Should we have a clear button to remove the configuration override?

Reply:
![image](https://github.com/DataDog/browser-sdk/assets/4342515/583e2538-234a-4d23-899e-3bd30ac31501)

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏in order to limit the number of clicks, what about having an edit and a clear button instead of opening a menu to then choose to edit or clear?